postman/scripts/sendMessageOnL1.ts

```diff
@@ -1,17 +1,10 @@
 import { BytesLike, ContractTransactionReceipt, Overrides, Wallet, JsonRpcProvider } from "ethers";
```

Comment:
@VGau should we move this script to sdk? I dun see how it's relevant to postman except as a handy tool for testing local postman

Reply:
This script is specific to the Postman so I think we should put it inside the SDK ? WDYT ?

Reply:
This script seems to me is agnostic to postman as it just sends messges on L1, so I'm thinking it should be moved to sdk for more generic usage, but up to you, I'm fine keeping it here, same for the script to send message  on L2